assets/stylesheets/common/wizard-admin.scss

```diff
@@ -711,3 +712,22 @@
   width: 80px;
   vertical-align: middle;
 }
+
+.btn.btn-pavilion-pro {
+  background: var(--pavilion-primary);
+  color: var(--pavilion-secondary);
+
+  .d-icon {
+    color: var(--pavilion-secondary);
+  }
+
+  &:hover,
+  &:focus {
+    background: darken($pavilionPrimary, 5%);
+
+    &[href],
+    svg.d-icon {
+      color: darken($pavilionSecondary, 10%);
```

Comment:
@keegangeorge  were there no `var(-- ` existing equivalents?

Reply:
You can't use SASS darken/lighten functions with CSS Custom properties so I used SCSS variables instead.

Reply:
Sure, understand that, just wondered if it's worth creating a variable in the standard pavilion set to cover these?  If not, please go ahead with your existing approach.